mods/ra/rules/infantry.yaml

```diff
@@ -725,3 +725,46 @@ Ant:
 	HitShape:
 		Type: Circle
 			Radius: 469
+
+FireAnt:
+	Inherits: Ant
+	Tooltip:
+		Name: Fire Ant
+		GenericVisibility: none
+	Mobile:
+		Speed: 80
+	Armament:
+		Weapon: AntFireball
+	Health:
+		HP: 70000
```

Comment:
To match the original, this should be 7500, but with
```
	Armor:
		Type: Heavy
```

That will make at least bunkers and grenades more effective (and rockets/cannons in later missions).